doc/rtd/explanation/failure_states.rst

```diff
@@ -0,0 +1,138 @@
+.. _failure_states:
```

Comment:
I'm not seeing any of the code blocks being rendered in this file. Nothing gets rendered at all in their place.